c_glib/Dockerfile

```diff
@@ -40,7 +40,7 @@ ENV ARROW_BUILD_TESTS=OFF \
     ARROW_BUILD_UTILITIES=OFF \
     ARROW_INSTALL_NAME_RPATH=OFF \
     LD_LIBRARY_PATH="${CONDA_PREFIX}/lib" \
-    PKG_CONFIG_PATH="${CONDA_PREFIX}/lib/pkgconfig" \
+    PKG_CONFIG_PATH="${CONDA_PREFIX}/lib/pkgconfig:/usr/lib/x86_64-linux-gnu/pkgconfig" \
```

Comment:
Is this needed?
The added path is included in system `pkg-config` path. So it'll be searched without this.

Reply:
I'm going to revert this for now